test/stat_tracker_test.rb

```diff
@@ -4,20 +4,20 @@
 
 class StatTrackerTest < MiniTest::Test
 
-  def test_it_exists
-    game_path = './data/games.csv'
-    team_path = './data/teams.csv'
-    game_teams_path = './data/game_teams.csv'
+  @game_path = './data/games.csv'
+  @team_path = './data/teams.csv'
+  @game_teams_path = './data/game_teams.csv'
 
-    locations = {
-      games: game_path,
-      teams: team_path,
-      game_teams: game_teams_path
-    }
+  @locations = {
+    games: @game_path,
+    teams: @team_path,
+    game_teams: @game_teams_path
+  }
 
-    stat_tracker = StatTracker.from_csv(locations)
+  @@stat_tracker ||= StatTracker.from_csv(@locations)
```

Comment:
By changing it so that stat_tracker is a class variable, this way @@stat_tracker can be called across all subclasses and still refers to the same object that is only instantiated once.

Reply:
Sweet beans. Glad we got this figured out!